ipapython/ipaldap.py

```diff
@@ -127,6 +127,12 @@ def ldap_initialize(uri, cacertfile=None):
             conn.set_option(ldap.OPT_X_TLS_CACERTFILE, cacertfile)
 
         # SSLv3 and SSLv2 are insecure
+        # OpenLDAP 2.4 sets minimum version with SSL_CTX_set_options(). The
+        # system-wide crypto-policies for TLS minimum version are applied
+        # with SSL_CTX_set_min_proto_version(). The set_option() call cannot
+        # not enable lower versions than allowed by crypto-policy, e.g.
```

Comment:
"cannot not" is a double negative in English so this reads "the set_option_call() can enable lower versions than allowed by crypto-policy. I think you need to drop the separate "not" based on the reset of the example, where one cannot override global policy within a library.

Reply:
@tiran could you please update the text based on @rcritten comments?